### PR TITLE
Filter dropdown levels to include only submittable levels for Rubrics

### PR DESCRIPTION
**Pull request by kobryan0619**
Currently when a Curriculum Writer create a new Rubric or edits a new rubric, they can pick any level in the lesson to connect the rubric to.  However, we want them to only connect it to a submittable level.  This change forces a Curriculum Writer to select a submittable level for their rubrics. 

**Old way** - the curriculum writer would see ALL the levels
image

**New way** - the curriculum writer would either see just the submittable levels or a warning to go back and make a level submittable if no submittable levels are available

image

image

## Testing story

Modified existing tests.

### CODE REVIEW

File: apps/src/lib/levelbuilder/rubrics/RubricEditor.jsx [hunk 1]
```diff
@@ -8,6 +8,7 @@ export default function RubricEditor({
   deleteLearningGoal,
   learningGoalList,
   updateLearningGoal,
+  disabled,
```
Comment (bethanyaconnor) [inline apps/src/lib/levelbuilder/rubrics/RubricEditor.jsx]: Is this needed? In the screenshot it looks like this component isn't loaded when there aren't submittable levels?

File: apps/src/lib/levelbuilder/rubrics/RubricsContainer.jsx [hunk 2]
```diff
@@ -204,10 +226,15 @@ RubricsContainer.propTypes = {
   unitName: PropTypes.string,
   lessonNumber: PropTypes.number,
   levels: PropTypes.arrayOf(
```
Comment (bethanyaconnor) [inline apps/src/lib/levelbuilder/rubrics/RubricsContainer.jsx]: Thoughts on combining this and the `hasSubmittableLevels` prop? ie pre-filtering `levels` to only contain the submittable levels then deducing `hasSubmittableLevels` from the length of that list?

File: apps/src/lib/levelbuilder/rubrics/RubricEditor.jsx (hunk 1 above)
Comment (kobryan0619) [reply apps/src/lib/levelbuilder/rubrics/RubricEditor.jsx]: Good point - this does mean this is no longer needed.  I'll make the change

File: apps/src/lib/levelbuilder/rubrics/RubricsContainer.jsx (hunk 2 above)
Comment (kobryan0619) [reply apps/src/lib/levelbuilder/rubrics/RubricsContainer.jsx]: I LOVE this!  I just refactored it and re-committed.

**Review by bethanyaconnor — APPROVED** [review]
Thanks for doing all these follow-ups!

**Pull request merged**